index.html

```diff
@@ -1319,7 +1333,8 @@
             <li>Let <em>A</em> be a new <code>PresentationAvailability</code>
             object with its <code>value</code> property set to
             <code>false</code> if the <a>list of available presentation
-            displays</a> is empty and <code>true</code> otherwise.
+            displays</a> is empty or non of them is a <a>compatible
+            presentation display</a>, <code>true</code> otherwise.
             </li>
             <li>Create a tuple <em>(A, presentationUrl, P)</em> and add it to
```

Comment:
Since we are always resolving P we may not need to keep it around to resolve it later; perhaps P can be dropped from the set of availability objects?  This cleanup can be made later.